end-to-end-tests/cypress/e2e/OSCAL/OSCAL_Edit_Tests.cy.js

```diff
@@ -43,78 +43,154 @@ describe('Test Editing System Security Plan Source', () => {
-    cy.contains(PARAM_VALUE_NEW).should('be.visible')
-  })
-})
+    cy.contains("button", COMPONENT_NAME).click();
+    cy.contains(PARAM_VALUE_NEW).should("be.visible");
+  });
+});
 
-describe('Test Adding New SSP Impl Req', () => {
+describe("Test Adding New SSP Impl Req", () => {
   // Store current SSP and reset after test
-  let origSspJson
+  let origSspJson;
   before(() => {
-    cy.getTestSspJson().then(result => origSspJson = result)
-  })
+    cy.getTestSspJson().then((result) => (origSspJson = result));
+  });
   after(() => {
-    cy.setTestSspJson(origSspJson)
-  })
+    cy.setTestSspJson(origSspJson);
+  });
 
-  it('Successfully Adds New SSP Impl Req', () => {
-    const CONTROL_ID = 'AU-2'
-    const CONTROL_NAME = 'Audit Events'
+  it("Successfully Adds New SSP Impl Req", () => {
+    const CONTROL_ID = "AU-2";
+    const CONTROL_NAME = "Audit Events";
     cy.navToSspViewer(SSP_TITLE_ORIG);
-    cy.contains('button', 'Add Control Implementation').click()
-    cy.contains('Select Control').click({force: true}).type(CONTROL_ID)
-    cy.contains(`${CONTROL_ID} ${CONTROL_NAME}`).click()
-    cy.intercept('GET', '/oscal/v1/system-security-plans/*').as('getSsp')
-    cy.get(`[aria-label="save-system-security-plan-control-implementation"]`).click()
-    cy.wait('@getSsp')
+    cy.contains("button", "Add Control Implementation").click();
+    cy.contains("Select Control").click({ force: true }).type(CONTROL_ID);
+    cy.contains(`${CONTROL_ID} ${CONTROL_NAME}`).click();
+    cy.intercept("GET", "/oscal/v1/system-security-plans/*").as("getSsp");
+    cy.get(
+      `[aria-label="save-system-security-plan-control-implementation"]`
+    ).click();
+    cy.wait("@getSsp");
     // Verify updated value
-    cy.contains('h2', CONTROL_NAME).should('be.visible')
-  })
-})
\ No newline at end of file
+    cy.contains("h2", CONTROL_NAME).should("be.visible");
+  });
+});
+
+describe("Test Editing Title Using Enter key", () => {
+  // Store current SSP and reset after test
+  let origSspJson;
+  before(() => {
+    cy.getTestSspJson().then((result) => (origSspJson = result));
+  });
+  afterEach(() => {
+    cy.setTestSspJson(origSspJson);
+  });
+
+  it("Successfully Edits Title with Enter", () => {
+    cy.navToSspViewer(SSP_TITLE_ORIG);
+    cy.get(`[aria-label="edit-system-security-plan-metadata-title"]`).click();
+    cy.get(`[data-testid="textField-system-security-plan-metadata-title"]`)
+    .click()
+    .clear()
+    .type("A Different SSP{enter}");
+    cy.contains("A Different SSP").should("be.visible");
+  });
+  
+  it("Successfully keeps title on Enter press", () => {
+    cy.navToSspViewer(SSP_TITLE_ORIG);
+    cy.get(`[aria-label="edit-system-security-plan-metadata-title"]`).click();
+    cy.get(`[data-testid="textField-system-security-plan-metadata-title"]`)
+      .click()
+      .clear()
+      .type("{enter}");
+    cy.contains(SSP_TITLE_ORIG);
+  });
+});
+
+describe("Cancel on ESC Keypress", () => {
```

Comment:
The wording for these tests are a bit confusing when reading the output. 

`Cancel on ESC Keypress ESC key cancels any changes`
`Cancel on ESC Keypress Enter key changes textfield value`

Is there a way to rename them so they are a bit more descriptive of what is being tested? I think its mostly the `describe` that makes it confusing.

Reply:
Sure thing, I can rename the tests. I was unsure if you have already renamed them in your request to change the test names in your other pull request. I can scrap the names have at the moment and reword them and make them more clear what the tests are doing.

Are the `Enter` and `Esc` test suites in the other pull request which your edit the test titles?

Reply:
If you look at #111, I have updated many of the tests made before your changes. As your `Editable Textfield` tests are not currently in `main`, I did not touch them